pyfstat/make_sfts.py

```diff
@@ -187,11 +188,12 @@ def __init__(
             Dictionary of timestamps (each key must refer to a detector),
             list of timestamps (`detectors` should be set),
             or comma-separated list of per-detector timestamps files
-            (simple text files, lines with <seconds> <nanoseconds>,
-            comments should use `%`, each time stamp gives the
-            start time of one SFT).
+            (simple text files,
+            comments should use `%`,
```

Comment:
```suggestion
            comments must use `%`,
```

We explicitly set that in `genfromtxt`. We may want to support `genfromtxt_kwargs` if you think it would be usable.
Probably it should, as it looks like we are moving away from octave?